python/metatomic_torch/metatomic/torch/ase_calculator.py

```diff
@@ -906,6 +951,53 @@ def _compute_ase_neighbors(atoms, options, dtype, device):
     )
 
 
+def _get_ase_input(
+    atoms: ase.Atoms,
+    quantity: str,
+    option: ModelOutput,
+    dtype: torch.dtype,
+    device: torch.device,
+) -> "TensorMap":
+    if quantity in ARRAY_PROPERTIES:
+        if len(ARRAY_PROPERTIES[quantity]) == 0:
+            raise NotImplementedError(
+                f"Though the property {quantity} is available in `ase`, it is "
+                "currently not supported by metatomic."
```

Comment:
IMO these should be made available as `ase::<name>`, using the same syntax as for custom outputs